vispy/visuals/_scalable_textures.py

```diff
@@ -258,9 +273,11 @@ def _scale_data_on_cpu(data, clim, copy=True):
         elif not copy and not np.issubdtype(data.dtype, np.floating):
             raise ValueError("Data must be of floating type for no copying to occur.")
 
-        if clim[0] != clim[1]:
+        if clim[0] == clim[1]:
+            pass  # Does not matter - clim range will be inf
```

Comment:
This if block seems...odd. I understand having it be explicit that this is why you don't care about it, but can't we just have the comment without the extra `if` statement. It makes the code harder to read in my opinion.

Reply:
I agree. I first did have code in this block, but I later removed it. Should have cleaned it up. Fixed!